src/main/java/com/jcabi/ssh/SSH.java

```diff
@@ -182,6 +193,17 @@ public SSH(final String adr, final int prt,
     }
 
     /**
+     * Constructor using SSHOptions object.
+     * @param options SSHOptions options object with connection properties
```

Comment:
@LinkedList could be 
`@param options SSHOptions object with connection properties`